api/api.py

```diff
@@ -116,8 +116,9 @@ def me(user):
                 twitter_user = twitter.get_user(user.twitter_username)
                 if not twitter_user:
                     return jsonify({'message': "Twitter profile not found!"}), 400
-                if twitter_user['created_at'] > (datetime.utcnow() - timedelta(days=app.config['TWITTER_USER_MIN_AGE_DAYS'])):
-                    return jsonify({'message': f"Twitter profile needs to be at least {app.config['TWITTER_USER_MIN_AGE_DAYS']} days old!"}), 400
+                if app.config['BASE_URL'] != "https://staging.plebeian.market":
```

Comment:
No, we can't do this, there has to be a better way without checking magic strings in the code.